state.go

```diff
@@ -200,9 +200,6 @@ func (s *State) PresenceAdd(guildID string, presence *Presence) error {
 			if presence.Status != "" {
 				guild.Presences[i].Status = presence.Status
 			}
-			if presence.Nick != "" {
-				guild.Presences[i].Nick = presence.Nick
```

Comment:
we no longer track nicknames?

Reply:
I logged a PresenceUpdate and noticed the field isn't being provided, same for "Since". Thats why I removed them from the struct and therefore also from the state.

Reply:
According to the discord api nick changes are only provided in GuildMemberUpdate events. Not in a presence update :thinking: